src/util/irep.h

```diff
@@ -43,6 +45,7 @@ typedef std::string irep_namet;
 // NOLINTNEXTLINE(readability/identifiers)
 typedef string_hash irep_id_hash;
 #endif
+typedef std::unordered_set<irep_idt, irep_id_hash> id_sett;
```

Comment:
It's probably a good idea to introduce a widely visible typedef, but I think this needs to happen in a separate PR: 1) Various places already define an `id_sett`; 2) those definitions don't always use the same types; 3) the type name should likely make clear that this is an unordered set of IDs.

Reply:
Okay, for this PR I'll just make it a local typedef, but I might try and make a widely visible one in another PR. For the record it's `std::unordered_set<irep_idt, irep_id_hash>` 9 times and `std::set<cpp_idt *>` 2 times. I like that `id_sett` is short, but I agree that you might assume it wasn't unordered.

Reply:
Here's a PR for you: https://github.com/diffblue/cbmc/pull/2075